source/quick-start.txt

```diff
@@ -14,6 +14,49 @@ Quick Start
 
 .. include:: /includes/quick-start/overview.rst
 
+Create a MongoDB Cluster
+------------------------
+
+Set Up a Free Tier Cluster in Atlas
```

Comment:
Moved a lot of this content directly into this page instead of an `includes` file, since it's not used anywhere else.